src/prototype_room_creepbuilder.js

```diff
@@ -122,7 +97,7 @@ Room.prototype.spawnCheckForCreate = function(creepsConfig) {
     var creep = this.memory.queue[0];
```

Comment:
for first propose, got the idea but you can't _.sortBy the array returned by _.map